### PR TITLE
Feature/mc 9976 bare metal compute offering inventory

### PR DESCRIPTION
**Pull request by m-orfanos**
Adds bare metal related documentation to API docs.

### CODE REVIEW

**Comment by m-orfanos** [comment]
Please ignore the first two commits. I sorted the attributes in the tables to make it easier for users to find what they're looking for, but it makes the PR bigger than it should.

File: source/includes/cloudstack/_bare_metal_instances.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Acquire a bare metal instance in an [environment](#administration-environments). This endpoint allows you to easily add [port forwarding rules](#port-forwarding-rules) and user data to the instance without doing additional API calls. This operation is analogous to the [create an instance](#cloudstack-create-an-instance) operation for virtual instances.
+
+Required | &nbsp;
+------ | -----------
+`name`<br/>*string* | Name of the newly created instance
+`templateId`<br/>*UUID* | The [template](#cloudstack-templates) to use for this instance.
+`computeOfferingId`<br/>*UUID* | The [compute offering](#cloudstack-compute-offerings) will determine the hardware specifications of your instance.
+`networkId`<br/>*UUID* | The [network](#cloudstack-networks) in which the instance will be created. If you don't have a network, it can be created through the [create network](#cloudstack-create-network) api.
+`sshKeyName`<br/>*string* | The name of the [SSH key](#cloudstack-ssh-keys) to use for this instance. If you don't have an SSH key registered, you can do so through this [api](#cloudstack-create-ssh-key).
+
```
Comment (y0geshdev) [inline source/includes/cloudstack/_bare_metal_instances.md:50]: We used to sort them alphabetically...no?

File: source/includes/cloudstack/_compute_offerings.md [hunk 2]
```diff
@@ -86,3 +99,5 @@ Attributes | &nbsp;
 `availableCpuCountValues`<br/>*Array[integer]* | The list of valid cpu counts when used in the [create instance operation](#cloudstack-create-an-instance). Only present for custom offerings
 `availableMemoryInMBValues`<br/>*Array[integer]* | The list of valid amounts of memory (in MB) that can be used in the [create instance operation](#cloudstack-create-an-instance). Only present for custom offerings
 `maxMemoryInMBToCpuRatio`<br/>*integer* | The maximum ratio of memory (in MB) to number of CPU of an [instance](#cloudstack-instances) created with this offering. Only present for custom offerings.
+`type`<br/>*enum* | The type of compute offering. Only present for bare metal compute offerings. Value is BareMetal.
+`availabilityCount`<br/>*integer* | The number of compute offerings available to acquire. Only present for bare metal compute offerings.
```
Comment (y0geshdev) [inline source/includes/cloudstack/_compute_offerings.md]: Can we sort these as well?

**Review by y0geshdev — CHANGES_REQUESTED** [review]

File: source/includes/cloudstack/_compute_offerings.md (hunk 2 above)
Comment (m-orfanos) [reply source/includes/cloudstack/_compute_offerings.md]: Done

File: source/includes/cloudstack/_bare_metal_instances.md (hunk 1 above)
Comment (m-orfanos) [reply source/includes/cloudstack/_bare_metal_instances.md:50]: Done

**Review by y0geshdev — APPROVED** [review]
Thanks for the changes.

**Pull request merged**